src/aes_key.c

```diff
@@ -1099,3 +1205,16 @@ static int aes_key_blob_is_pkey_extractable(struct zpc_aes_key *aes_key, const u
 
 	return 0;
 }
+
+static void aes_key_add_ep11_header(struct zpc_aes_key *aes_key)
+{
+	struct ep11kblob_header *ep11hdr;
+
+	memset(aes_key->cur.sec, 0, sizeof(struct ep11kblob_header));
+	memmove(aes_key->cur.sec + sizeof(struct ep11kblob_header), aes_key->cur.sec, aes_key->cur.seclen);
```

Comment:
Are you sure that the size of the buffer is large enough here?

Reply:
Yes, the sec buffer has MAX_AESKEYBLOBSIZE = 512 bytes, so plenty of space.

Reply:
I meant to add a check into the code, and not just trust that it is enough :-)

Reply:
OK fixed ;-)